README.md

```diff
@@ -13,8 +13,10 @@ cd plotIt/
 
 # Initialize the git remotes
 source firstsetup.sh 
-# Load the proper environment (a la CMSSW_7_4_10)
-source setup_sl6_env.sh
```

Comment:
Why remove this? plotIt works outside of CMS as well... But it's true that the `setup_sl6_env.sh` file is not up-to-date anyway.